_assets/patches/geth/0016-geth-181-whisperv6-peer-race-cond-fix.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/whisper/whisperv6/peer.go b/whisper/whisperv6/peer.go
```

Comment:
this PR was merged, do you want to sync with current master?

Reply:
We can go either way. I marked the patch as 1.8.1-only, so in any case the idea is to remove it for 1.8.2. I just didn't sync to master to stay with a released version of geth.